pyproject.toml

```diff
@@ -16,9 +16,9 @@ dependencies = [
 ]
 requires-python = ">=3.8"
 readme = "README.rst"
-license = {file = "LICENSE"}
+license = "BSD-3-Clause"
+license-files = ["LICENSE"]
```

Comment:
Should the `build-system.requires` be bumped for this one to `>=3.11`?
https://github.com/pypa/flit/blob/2818f41fb2a6aedfd264c175ed3ca1434f39674b/pyproject.toml#L1-L3

Reply:
Good point, yes!